frontend/web/project/api.js

```diff
@@ -91,7 +91,7 @@ global.API = {
     }
 
     flagsmith
-      .identify(user.email, {
+      .identify(user.id, {
```

Comment:
I'd argue maybe this should be some concatenated string to avoid interspersion with the previous email based identities. Something like the following perhaps? 

```suggestion
      .identify(`user-${user.id}`, {
```

Reply:
Yep, fair enough. I do still think that using a string concatenation like this is still a good idea, even if just for aesthetics.

Reply:
I think your suggestion around having a user guid would be the best, if we want to do that I'll hold off on this PR until we do that @matthewelwell ?